pkg/network/http/shared_libraries.go

```diff
@@ -178,6 +197,33 @@ func (w *soWatcher) sync() {
 	}
 }
 
+func (w *soWatcher) checkProcessDone() (updated bool) {
+	updated = false
+	for libpath, registration := range w.registry.byPath {
+		for pid := range registration.pids {
+			/* check if the process is alive */
+			var err error
+			var fi os.FileInfo
+			procPidPath := util.GetProcRoot() + string(os.PathSeparator) + strconv.Itoa(pid)
```

Comment:
can move out from the loop the concatenation of `util.GetProcRoot() + string(os.PathSeparator)` to spare string operations